### DISCUSSION TITLE
How to read and write to iceberg table stored in Azure Blob Storage?

### DISCUSSION THREAD

**Discussion by nikola-arcadis** [apache/doris#39522] [A - General / Q&A]
Does Doris v3 have support to read and write to iceberg table having data stored in Azure Blob Storage?
I can find s3 examples in the documentation but how to connect to Azure Blob Storage?
Also, by separating compute and storage in Doris is it possible to configure Doris to store its own table data in Azure Blob Storage?

**Comment by ixzc** [comment]
not support, at present. but we have a plane to do this

**Reply by nikola-arcadis** [reply]
Thanks.  We are using s3 proxy for now. Azure support is greatly appreciated.
